interfaces/DepositSecurityModule.json

```diff
@@ -1,600 +1,613 @@
 [
-    {
```

Comment:
Maybe we should not update DSM interface yet

Reply:
Agree